pkg/cmd/set_replication.go

```diff
@@ -89,7 +89,14 @@ func NewCmdVolSyncSetReplication(streams genericclioptions.IOStreams) *cobra.Com
 
 func (o *FinalizeOptions) bindFlags(cmd *cobra.Command, v *viper.Viper) error {
 	flags := cmd.Flags()
+
+	if o.sourceName == "" {
+		o.sourceName, _ = v.Get("source-replication-name").(string)
+	}
 	flags.StringVar(&o.sourceName, "source-replication-name", o.sourceName, "name of ReplicationSource (default '<source-ns>-source')")
```

Comment:
It looks like you're trying to update the default value for the flags. I think a better approach would be to ensure the flag names are standardized across the different operations.

For example, the current issue of `start-replication` vs `set-replication` looks like it comes down to the flags being named differently (and not documented).
`start-replication` seems to take "source-name" (L126 of create_replication.go) whereas `set-replication` takes "source-replication-name".

My suggestion would be to take a look at the flag names and adjust them so that they are consistent across the different sub-commands. It would also be beneficial to ensure consistent naming of the struct fields, too. (i.e. `o.SourceName` vs `o.Name`)

A good review and cleanup could probably take care of #10 and #49 as well.

Reply:
Makes sense. I too came across these differences but was not sure whether to take of those changes through this PR. 
I will do the needful.